### DISCUSSION TITLE
Como vocês emitem nota fiscal para trabalhos na Upwork?

### DISCUSSION THREAD

**Discussion by ghost** [DyegoCosta/trabalhando-remoto#206] [Q&A]
Não existe quase nada de informação na internet para quem trabalha pela Upwork e pretende receber como PJ e emitir notas. Sendo assim, escrevi um relato da minha experiência, pode ajudar alguém que está na mesma. Para ir direto ao assunto, é só rolar a tela até o TL;DR no final do texto.

Comecei a trabalhar pela Upwork há alguns meses, e como o rendimento foi aumentando, chegou a hora de abrir uma empresa. Escolhi a Contabilizei para fazer a contabilidade e, sinceramente, estou arrependido. Eles não sabem informar nada, você entra em contato com o suporte e te enviam um copia e cola de algo nada a ver com a pergunta. Contratei o plano de 149 reais, que inclui suporte via WhatsApp, e consegui mais informações com o pré-venda da Agilize do que com eles — também tive que pagar 300 reais pela consultoria de um contador que outro brasileiro que trabalha pela Upwork me indicou. Então, minha primeira consideração é, se for trabalhar pela Upwork, escolha outra empresa de contabilidade — o pessoal da Contabilizei nem sabe o que é Upwork.

Enfim, vamos lá. Entrei em contato com a Contabilizei para saber em nome de quem eu deveria emitir a nota, se da Upwork ou do cliente. Segundo eles — assim como a Agilize e o contador particular — a nota deve ser emitida no nome do cliente. Acontece que a Upwork não fornece os dados completos do cliente, apenas o nome da pessoa física, o nome da pessoa jurídica ou o nome fantasia — um ou outro, se houver — e o país. Entrei em contato com a Upwork e, segundo eles, não informam outros dados por questões de privacidade. O cliente escolhe se o endereço dele pode ser incluído na invoice ou não e, no fim das contas, de todos os serviços que prestei, nenhum possui o endereço na invoice.

Fui então procurar por outras pessoas na mesma situação. Descobri nos fóruns da Upwork que diversos freelancers que residem na Europa passam pelo mesmo problema. Pelo que entendi, a Upwork se limita a recomendar que o profissional entre em contato com o cliente e peça o endereço para ele. Enfim, é um constrangimento desnecessário e, sinceramente, já é difícil conseguir trabalhos por lá; não tenho interesse em colocar mais essa barreira entre eu e o cliente.

Voltei ao atendimento da Contabilizei, e a atendente simplesmente me largou na mão. A posição da empresa foi a de que não existe uma situação assim, que ela nunca tinha visto uma invoice igual a da Upwork, e que as informações de todo mundo são públicas e eu deveria procurar pelo endereço do cliente na internet — sendo que o cliente é uma pessoa física, mas enfim, não queria entrar num debate com ela sobre isso. Nesse ponto já estava quase emitindo uma nota em nome da Upwork mesmo — já que recebia por eles e possuía todos os dados.

Decidi trocar de contabilidade, e foi aí que entrei em contato com a Agilize. Expliquei toda a situação e conseguiram esclarecer todas as minhas dúvidas em questão de uma ou duas horas — acho que vou mudar para eles assim que passar o prazo da multa rescisória, me parece que eles estão muito mais preparados para atender profissionais que trabalham para o exterior. Resumindo toda a história, eles sabem o que é a Upwork e, segundo eles, a prefeitura de São Paulo permite a emissão de notas apenas com o nome do cliente e o país. Voltei para o painel da Contabilizei, achando que ia finalmente emitir a bendita nota fiscal, mas que nada. O emissor de notas deles tem campo para o endereço completo, apartamento, cidade, tudo, menos o país. Em contato com o suporte, novamente tive que explicar toda a situação, e novamente a mesma resposta: você precisa do endereço completo do cliente.

Foi aí que consegui o contato de um contador que presta serviços para pessoas que trabalham para o exterior, através de outro brasileiro que trabalha para a Upwork. Marquei uma consulta e, basicamente, ele me deu a mesma resposta que a Agilize. Segundo ele, o serviço prestado pela Upwork é uma grande área cinza aqui no Brasil. No contrato eles se apresentam apenas como um classificados de emprego, mas por realizarem os pagamentos, terem o poder de decidir se o freelancer vai receber ou não, o poder de bloquear seu perfil, etc., legalmente eles seriam o cliente. Na dúvida, ele sugere seguir o contrato da Upwork, considerando eles apenas como mediadores do serviço, mas que essa interpretação ficaria a critério do fiscal da receita que porventura venha solicitar esclarecimentos sobre sua atividade.

Mais uma vez entrei em contato com a Contabilizei, e foi um quase um parto para conseguir uma posição final deles. Passei tudo o que a Agilize e o contador me informaram, e a atendente simplesmente se limitou a dizer para tentar emitir a nota só com o nome do cliente e o país. Se a prefeitura autorizasse a emissão, então estava autorizado, que a única preocupação do governo era em receber o imposto — só que a minha é não ter que enfrentar uma auditoria por incompetência deles.

## TL;DR

Depois de tanta dor de cabeça, finalmente consegui emitir uma nota pela Contabilizei para um serviço prestado pela Upwork. No campo nome, informei o nome da empresa, e no campo logradouro, coloquei o país. No campo de descrição, perguntei para a atendente se deveria colocar alguma observação ali, ela se limitou a dizer que eu poderia colocar o que eu quisesse. Nenhuma informação concreta. Sendo assim, copiei e colei o modelo que eles apresentam nas páginas de ajuda:

> 7 - Em Descrição do serviço, faça um resumo do serviço prestado.
> 
> Atenção: indique na nota fiscal que é uma exportação de serviços, informando o número da 'invoice' (temos um modelo pronto para você, veja aqui) e a moeda em que o serviço será pago.
> 
> Por exemplo: “Desenvolvimento de sistema - 98h - 32 USD a hora - 20.000,00  USD. Cotação do dia R$ 5,00. Invoice Nº 0001”.

- Alguém emite nota de outra forma para os serviços prestados pela Upwork?
- Existe uma forma mais correta para preencher a nota?
- Como vocês lidam com o fato de não haver um contrato assinado pelo cliente? Recebo pela Husky, segundo eles só o invoice da Upwork já vale, mas sei de gente que teve o valor bloqueado no Remessa Online por não ter um contrato de trabalho como comprovação da origem do valor.

**Comment by Vittalev** [comment]
Oi @cdtoledo , comecei a trabalhar pelo Upwork tb e gostaria de saber como vc faz pra receber o valor via PJ de lá. Tem que abrir uma conta em dólar? A partir dessa conta em dólar envia pra conta PJ em reais no Brasil? Se puder me dar um help agradeço mt. Abç!

**Reply by ghost** [reply]
Cara, atualmente estou recebendo pela Husky para transferir para uma conta PJ no Brasil. Se você receber em uma conta lá fora, você perde a isenção do IOF. Na verdade, recebendo pela Husky, acaba ficando elas por elas, por conta da taxa de 1% deles. De qualquer modo, se for receber em uma conta em dólar, esta tem que ser PJ. Se transferir para uma conta PF em dólar primeiro, e depois para PJ, vai caracterizar como aporte de capital, você terá que pagar o IRPF em cima do valor, e perderá todos os benefícios da PJ.

Fuja do BS2 a todo custo! Há alguns meses apenas eles ofereciam conta PJ em dólar. Parece que o C6 agora oferece essa opção, mas não cheguei a testar. O Payoneer é a maior furada também. Tinha a conta normal deles e usei por anos. Solicitei a PJ, foi aprovada, e comecei a receber pagamentos por ela. Quando fui sacar, solicitaram o preenchimento de um questionário, e não me deram mais retorno. O suporte deles me respondia depois de dias, e sempre com respostas automatizadas, sem qualquer solução. Comecei a reclamar em diversos canais, simplesmente bloquearam minha conta, e estou com três semanas de pagamento presos lá há meses, sem qualquer retorno da empresa. Até onde consegui entender, eles não tem autorização para operar contas PJ, o dinheiro algumas vezes trava no meio da transferência, e eles somem.

Se for utilizar a Husky e precisar de um cupom de 1%, me avisa que te envio!
